### PR TITLE
check if account_server_info is 0AC4 not 0069

### PR DESCRIPTION
**Pull request by sctnightcore**
Tested with kRO_RagexeRE_2018_06_21a
image

### CODE REVIEW

File: src/Base/Ragnarok/AccountServer.pm [hunk 1]
```diff
@@ -94,22 +94,78 @@ sub master_login {
...
+			} else {
+				push @servers, {
+					ip => $ip,
+					port => $charServer->getPort,
+					name => $charServer->getName,
+					users => $charServer->getPlayersCount,
+					display => 5, # don't show number of players
+					ip_port => $ip . ':' . $charServer->getPort,
+				};
+			}
```
Comment (ya4ept) [inline src/Base/Ragnarok/AccountServer.pm:146]: I think we can do it better, like this:
```perl
			if ($clientDate > 20170315) {
				push @servers, {
					ip => $ip,
					port => $charServer->getPort,
					name => $charServer->getName,
					users => $charServer->getPlayersCount,
					display => 5, # don't show number of players
					ip_port => $ip . ':' . $charServer->getPort,
				};
			} else {
				push @servers, {
					ip => $ip,
					port => $charServer->getPort,
					name => $charServer->getName,
					users => $charServer->getPlayersCount,
					display => 5, # don't show number of players
					state => 0,
					property => 0,
					unknown => 0,
					ip_port => $ip . ':' . $charServer->getPort,
				};
			}
```

**Comment by ya4ept** [comment]
Maybe we should just add this package to the appropriate serverType like in bRO?
```perl
	my %handlers = qw(
		account_server_info 0AC4
	);
```

File: src/Base/Ragnarok/AccountServer.pm [hunk 2]
```diff
@@ -94,22 +94,78 @@ sub master_login {
...
+				} else {
+					push @servers, {
+						ip => $ip,
+						port => $charServer->getPort,
+						name => $charServer->getName,
+						users => $charServer->getPlayersCount,
+						display => 5, # don't show number of players
+						state => 0,
+						property => 0,
+						unknown => 0,
```
Comment (ya4ept) [inline src/Base/Ragnarok/AccountServer.pm:132]: what is "unknown"?
**Network\Receive.pm:**
```perl
	} elsif ($args->{switch} eq '0AC4' || $args->{switch} eq '0B07') { # kRO Zero 2017, kRO ST 201703+, vRO 2021
		$server_info = {
			len => 160,
			types => 'a4 v Z20 v3 a128',
			keys => [qw(ip port name users state property ip_port)],
		};
```

**Comment by alisonrag** [comment]
> Maybe we should just add this package to the appropriate serverType like in bRO?
> 
> ```perl
> 	my %handlers = qw(
> 		account_server_info 0AC4
> 	);
> ```

i was adding these packets to servertype.pm to add check in xkore2, i lost the branch where i was working on this =/

but was something like that:
`if ($self->{packet_lut}{account_server_info} eq '0AC4') {`

**Comment by sctnightcore** [comment]
waiting for hacktoberfest
https://hacktoberfest.digitalocean.com/

**Pull request closed without merging**